boxoffice/static/js/models/util.js

```diff
@@ -21,23 +21,31 @@ export const Util = {
   }
 }
 
-export const fetch = function(config){
+export const fetch = function (config){
   return $.ajax({
     url: config.url,
     dataType: 'json'
   });
 }
 
-export const post = function(config){
+export const post = function (config){
   return $.ajax({
     url: config.url,
     type: 'POST',
     dataType: 'json'
   });
 }
 
-export const scrollToElement = function(element, speed=500) {
+export const scrollToElement = function (element, speed=500) {
   $('html,body').animate({
     scrollTop: $(element).offset().top
   }, speed);
 }
+
+export const completePageLoad = function (newTitle) {
```

Comment:
Can you please add a comment to describe what this function does? An example would help.